src/components/frame/TokenActivation/StakesTab/StakesTab.css

```diff
@@ -0,0 +1,89 @@
+.stakesContainer {
+  height: calc(551px - 42px);
```

Comment:
If both values aren't dynamic, why not just use `509px` without the `calc()`? :monocle_face:

Reply:
Sounds good 👍